src/main/java/org/breskul/util/AnnotationsHelper.java

```diff
@@ -0,0 +1,97 @@
+package org.breskul.util;
+
+import org.breskul.connectivity.annotation.Column;
+import org.breskul.connectivity.annotation.Id;
+import org.breskul.connectivity.annotation.Table;
+import org.breskul.exception.BoboException;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AnnotationsHelper {
+    public static String getTableName(Object object) {
```

Comment:
Why do you need the instance of class here? I think,  it would be enough to have information only about the class

Reply:
Good point! Fixed for all Helper class.